conf/httpd.conf.d/httpd.admin

```diff
@@ -103,6 +103,8 @@ if (defined($management_network->{'Tip'}) && $management_network->{'Tip'} ne '')
 $User = "pf";
 $Group = "pf";
 
+$PerlChildInitHandler = "pf::WebAPI::ChildInitHandler";
```

Comment:
Without the PerlChildInitHandler pf::WebAPI::InitHandler::child_init
The socket will be shared across multiple processes.
It will work fine in a unstressed environment.
Underload it could cause a problem.